model/src/main/proto/question.proto

```diff
@@ -82,14 +82,20 @@ message AnsweredQuestionOutcome {
   bool is_correct_answer = 2;
 }
 
+// A user's grade for a practice session represented as a fraction.
+message FractionGrade {
```

Comment:
This is a bit confusing since fractions can't be a real value divided by a real value. For what reason do we want to keep a numerator and denominator here?

Reply:
I believe Chantel wants to display the score for each skill as a fraction (so we need both numerator and denominator), and Oppia web is calculating scores as real values instead of integers, so I did the same for consistency. Do we want to multiply everything by a factor of 10 so we have integers instead?

Reply:
I think that's a worthwhile simplification. I see the current implementation and I find it much easier to follow without the floating point bits.

That being said, a few thoughts:
1. If you're multiplying everything by '10', we should make that '10' a Dagger constant since it's now a magic value (e.g. something like 'max points per question' or reuse the existing constant)
2. 'numerator' and 'denominator' are still really unclear. Could we just use the same terms that are being used elsewhere? E.g. 'pointsReceived' & 'totalPointsAvailable', respectively?

Reply:
Done, thanks!

Reply:
Following up on this--can these values be ints now that everything is being scaled? You could always perform the final decimal calculation in the UI and keep things ints at the domain layer to simplify management.

Reply:
I kept it as a double because I think Chantel may want to display the scores as a fraction in the UI (e.g. "you got score x / y"), so in that case I thought it would be better to calculate the final values to be consistent with the way Oppia web does it  (which may not be integers). To do the calculation, we would also need access to the multiplier constant, which is in the domain module. I thought it made more sense to complete the calculations here before sending off the final score (ready for display) to the app module. Let me know what you think!